packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -65,30 +68,37 @@ export class Runner {
 
         await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
 
+        let scanSucceeded: boolean;
         try {
             await this.webDriverTask.launch();
-            await this.scan(pageScanResult, scanMetadata.url);
-            this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
```

Comment:
I'm wondering if "completedScanRequests" value is always 1, what value does it add if we already know that it is a "ScanRequestCompleted" event?